streams/streams-scala/src/test/scala/org/apache/kafka/streams/scala/kstream/StreamJoinedTest.scala

```diff
@@ -16,22 +16,36 @@
  */
 package org.apache.kafka.streams.scala.kstream
 
-import org.apache.kafka.streams.kstream.internals.StreamJoinedInternal
+import org.apache.kafka.streams.kstream.internals.{InternalStreamsBuilder, StreamJoinedInternal}
+import org.apache.kafka.streams.processor.internals.InternalTopologyBuilder
 import org.apache.kafka.streams.scala.serialization.Serdes
 import org.apache.kafka.streams.scala.serialization.Serdes._
 import org.apache.kafka.streams.state.Stores
+import org.easymock.EasyMock
+import org.easymock.EasyMock.{createMock, replay}
 import org.junit.jupiter.api.Assertions.assertEquals
-import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.{BeforeEach, Test}
 
 import java.time.Duration
 
 class StreamJoinedTest {
 
+  val builder: InternalStreamsBuilder = createMock(classOf[InternalStreamsBuilder])
+  val topoBuilder: InternalTopologyBuilder = createMock(classOf[InternalTopologyBuilder])
+
+  @BeforeEach
+  def before(): Unit = {
+    EasyMock.expect(builder.internalTopologyBuilder()).andReturn(topoBuilder);
```

Comment:
We should be using `mockito` for new tests. A large number of tests have been converted (a huge amount of work) - this is adding more conversion work. cc @divijvaidya

Reply:
Great news! I just matched what was already in this file, but for future reference it's alright to mix EasyMock/Mockito in the same test files?

Reply:
Ah it appears my memory is hazy - I'm not sure why I used EasyMock here...

Reply:
We avoid mixing, but this file didn't have any EasyMock before (as you said in your second comment). Feel free so submit a follow-up that changes it to Mockito. :)

Reply:
Apologies, I should have caught this. I'm guessing Almog copied it from a java class that hadn't been converted yet, even though the scala test class was. Anyways thanks for the notice Ismael (Almog just ping me when you have a followup PR for this)

Reply:
https://github.com/apache/kafka/pull/15424